front/src/applications/stdcm/components/StdcmHeader.tsx

```diff
@@ -18,12 +22,14 @@ const StdcmHeader = ({ isDebugMode, onDebugModeToggle }: StdcmHeaderProps) => {
           {t('notificationTitle')}
         </span>
       </div>
+
       <div className="stdcm-header_debug">
         <button
           type="button"
           aria-label="stdcm-debug"
           className={cx({ 'debug-on': isDebugMode, 'debug-off': !isDebugMode })}
           onClick={() => onDebugModeToggle(!isDebugMode)}
+          aria-disabled={!isSuperUser}
```

Comment:
I think we can just use the `disabled` attribute here, since this is a `button` element.